baobab/bnn_priors/models.py

```diff
@@ -0,0 +1,292 @@
+    .. [1] D’Onofrio, Mauro, et al. 
+    "On the Origin of the Fundamental Plane and Faber–Jackson Relations: Implications for the Star Formation Problem." 
+    The Astrophysical Journal 838.2 (2017): 163.
+
+    .. [2] Nigoche-Netro, A., et al. 
+    "The Faber-Jackson relation for early-type galaxies: dependence on the magnitude range." 
+    Astronomy & Astrophysics 516 (2010): A96.
+
+    """
+    log_L_V = slope*np.log10(vel_disp) + intercept
+    return log_L_V
+
+def size_from_fundamental_plane(vel_disp, m_V, a=1.4335, b=0.3150, c=-8.8979):
+    """Evaluate the size expected from the Fundamental Plane (FP) relation
+    for a given velocity dispersion and V-band apparent magnitude
+
+    Parameters
+    ----------
+    vel_disp : float
+        the velocity dispersion in km/s
+    m_V : float
+        the apparent V-band magnitude
+    slope : float
+        slope of the log(L_V/L_solar) vs. log(vel_disp) relation
+    intercept : float
+        intercept of the log(L_V/L_solar) vs. log(vel_disp) relation
+
+    Returns
+    -------
+    float
+        the effective radius in kpc
+
+    Note
+    ----
+    The default values of slope and intercept are taken from the r-band orthogonal fit
+    on SDSS DR4. See Table 2 of [1]_.
+    V-band has rest-frame wavelength range 480nm ~ 650nm.
+
+    References
+    ----------
+    .. [1] Hyde, Joseph B., and Mariangela Bernardi. 
+    "The luminosity and stellar mass Fundamental Plane of early-type galaxies." 
+    Monthly Notices of the Royal Astronomical Society 396.2 (2009): 1171-1185.
+
+    """
+    log_R_eff = a*np.log10(vel_disp) + b*m_V + c
+    R_eff = 10**log_R_eff
+    return R_eff
+
+def gamma_from_size_relation(R_eff, slope=-0.41, intercept=0.39, delta_slope=0.12, delta_intercept=0.10, intrinsic_scatter=0.14):
+    """Evaluate the power-law slope of the mass profile using the empirical correlation derived 
+    from the SLACS lens galaxy sample with a given effective radius
+
+    Parameters
+    ----------
+    R_eff : float
+        the effective radius in kpc
+
+    Note
+    ----
+    See Table 4 of [1]_ for the default fit values used.
+
+    References
+    ----------
+    .. [1] Auger, M. W., et al. 
+    "The Sloan Lens ACS Survey. X. Stellar, dynamical, and total mass correlations of massive early-type galaxies." 
+    The Astrophysical Journal 724.1 (2010): 511.
+
+    """
+    log_R_eff = np.log10(R_eff)
+    gamma_minus_2 = log_R_eff*slope + intercept
+    gamma = gamma_minus_2 + 2.0
+    gamma_sig = (intrinsic_scatter**2.0 + np.abs(log_R_eff)*delta_slope**2.0 + delta_intercept**2.0)**0.5
+    scatter = np.random.randn()*gamma_sig
+    return gamma + scatter
+
+def axis_ratio_from_SDSS(vel_disp, A=0.38, B=5.7*1.e-4, truncate=0.2):
+    """Sample (one minus) the axis ratio of the lens galaxy from the Rayleigh distribution with scale
+    that depends on velocity dispersion
+
+    Parameters
+    ----------
+    vel_disp : float
+        velocity dispersion in km/s
```

Comment:
same comment as for function above. The SDSS axis ratio is defined by your default input statements and thus is not inherently part of the function itself (which is more general).

Reply:
Updated the docstring.